packages/block-editor/src/components/block-mover/index.js

```diff
@@ -56,12 +56,12 @@ export class BlockMover extends Component {
 		// to an unfocused state (body as active element) without firing blur on,
 		// the rendering parent, leaving it unable to react to focus out.
 		return (
-			<div className={ classnames( 'editor-block-mover block-editor-block-mover', { 'is-visible': isFocused || ! isHidden } ) }>
+			<div className={ classnames( 'editor-block-mover block-editor-block-mover', { 'is-visible': isFocused || ! isHidden, 'is-horizontal': orientation === 'horizontal' } ) }>
 				<IconButton
 					className="editor-block-mover__control block-editor-block-mover__control"
 					onClick={ isFirst ? null : onMoveUp }
-					icon={ upArrow }
-					label={ __( 'Move up' ) }
+					icon={ orientation === 'horizontal' ? leftArrow : upArrow }
+					label={ orientation === 'horizontal' ? __( 'Move left' ) : __( 'Move up' ) }
 					aria-describedby={ `block-editor-block-mover__up-description-${ instanceId }` }
```

Comment:
Besides the aria-label the movers contain an aria description referred in aria-describedby. The aria description is generated using the function getBlockMoverDescription this function in parts of the test refers "Move down/up" I think we should make this function aware if the movers are horizontal or vertical.